src/CKAN.jl

```diff
@@ -0,0 +1,56 @@
+using JSON
+
+immutable CKAN <: DataRepo
+end
+
+base_url(::CKAN) = "https://datadryad.org/mn/object/http://dx.doi.org/"
+
+function description(repo::CKAN, mainpage)
+    j = JSON.parse(text_only(mainpage.root))
```

Comment:
No single letter variables.
They are impossible to ctrl+f for.
I'd call this `data` or something

Reply:
Probably even

```
data = JSON.parse(text_only(mainpage.root))["result"]
```

So you can avoind indexing into `result` each time it is used below